### DISCUSSION TITLE
Is the docker stopped including `ebpf` plugin?

### DISCUSSION THREAD

**Discussion by stavros-k** [netdata/netdata#12800] [Q&A] [answered]
Hey, I'm quite new in netdata (Started messing around the couple last days).
I was trying to enable ZFS Filesystem metrics, with the EBPF plugin, but realized that it was missing from `/usr/libexec/netdata/plugins.d/`

I was using the `latest` tag which results in version `v1.34.0-69-nightly`.
I tried to use `1.31.0` `1.32.0` and `1.33.0`, which all of them **included** the `ebpf.plugin`.

I also tried `1.34` but it didn't. (`1.34.0` and `1.34.1` was failing to pull)

I've searched github around but didn't find anything clear if it was removed or something.

Didn't want to make an issue as I'm not sure if I'm missing something here, being a new guy :P

**Comment by Ferroin** [comment, marked as answer]
Last I knew the eBPF plugin has never worked properly when run in a Docker container, and I think we actually did change things between 1.33.1 and 1.34.0 to stop trying to ship it in our Docker images as a result, but I do not recall for certain. I may, however, be missing something here myself.

**Reply by stavros-k** [reply]
I see, well even if it wasn't supposed to be removed now, not working properly makes it not worth to try bringing it back!

I'll if there are any other options on the plugins to collect zfs metrics.

Thanks!
